articles/tables.md

```diff
@@ -164,8 +162,8 @@ If you upload a file, the Web interface will automatically detect the table sche
 The Table() function takes two arguments, a schema object and data in some form, which can be:
 
 - a path to a CSV file
-- a [Pandas](http://pandas.pydata.org/){:target="_blank"} [DataFrame](http://pandas.pydata.org/pandas-docs/stable/api.html#dataframe){:target="_blank"} (This is a Python-only feature)
-- a `RowSet` object
+- a dataframe
```

Comment:
This is a Python specific section, yes? Python doesn't have data frames (other than in Pandas).